Gruntfile.js

```diff
@@ -96,4 +113,17 @@ module.exports = function (grunt) {
         'sass'
     ]);
 
+    grunt.registerTask('lint', 'Run the JS linters.', [
+        'jshint',
+        'jscs'
+    ]);
+
+    grunt.registerTask('test', 'Run the tests.', function (env) {
+         'lint'
+    });
+
+    grunt.registerTask('travis', 'Run the tests in Travis', [
+        'test:travis'
+    ]);
```

Comment:
I don't think you need `travis` and `test:travis`, just `grunt test` in travis will do. This repo is unlikely to need to have separate local and travis tests